cla_backend/apps/call_centre/admin/__init__.py

```diff
@@ -7,14 +7,34 @@
 from ..models import Operator, Caseworker, Organisation
 
 
+class OrganisationListFilter(admin.SimpleListFilter):
+    title = "Organisation"
+    parameter_name = "organisation"
+
+    def lookups(self, request, model_admin):
```

Comment:
I remembered what I was trying to describe earlier!
It's `queryset.values()`

So you should be able to replace the construction of the organisations list with `Organisation.objects.values('id', 'name')`
You'll need to convert that to a list (`oraganisations = list(organisations)`) before appending to it, I think